packages/avado/dappnode_package.json

```diff
@@ -17,18 +17,8 @@
       "3001:3001/tcp"
     ],
     "path": "hopr.avado.dnp.dappnode.eth_0.100.0.tar.xz",
-<<<<<<< HEAD
```

Comment:
looks like a merge issue

Reply:
Yeah, the base branch is quite shitty, it's not master, so I just fixed them here.